src/main/webapp/WEB-INF/jsp/restaurant.jsp

```diff
@@ -9,6 +9,7 @@
   <title>Restaurant</title>
   <script src="/js/util.js"></script>
   <script src="/js/restaurant-page.js"></script>
+  <link href="/css/deal.css" rel="stylesheet" />
```

Comment:
I moved the css into styles.css in #89. I'm not sure if it's better to keep it in deal.css, or move it into the global style sheet.

Reply:
ok I'll keep the file in this PR and merge the changes after 89 is merged into master